app/src/main/res/layout-land/pin_password_activity.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.text.InputType" />
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.PinPasswordViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/oppiaLightGreen"
+    android:paddingTop="48dp"
+    android:paddingBottom="76dp">
+
+    <TextView
+      android:id="@+id/hello_text_view"
```

Comment:
Are we supposed to rename these textviews specific to file name?

Reply:
As per @rt4914 we are having dedicated people working on the naming convention hence he told me to skip that.

Reply:
Ack, because if we change naming in this PR, it will unnecessarily become big.